### ISSUE TITLE
Show current (super) section title

### ISSUE THREAD

**Issue by camillobruni** [bterlson/ecmarkup#60]
While browsing the spec I typically end up looking for something like "[[Call]]" which exists in multiple places (Functions, Proxies...). In this case it is not always immediately clear in which section you are.

It would be nice to highlight the current super-section somehow. One option would be to show it in the expanded TOC, another one might be using fixed Breadcrumps at the top of the specs

**Comment by bterlson** [comment]
A "reveal in toc" button/link for clauses would be really handy and probably satisfy your use case?

**Comment by domenic** [comment]
@jorendorff's version used position: sticky to implement this in Firefox I believe.

**Comment by bterlson** [comment]
@Domenic I can't see what you're referring to, can you explain what functionality you're referring to?

**Comment by domenic** [comment]
@bterlson open Firefox, and resize your window so it is vertically small. Then go to e.g. https://people.mozilla.org/~jorendorff/es6-draft.html#sec-functionallocate and scroll down a bit. You can see how the section header sticks to the top, due to the `position: sticky` heading. This could be extended up to all sectioning levels, I believe.

**Comment by camillobruni** [comment]
@dmenic that would be exactly what I imagined.

**Comment by jorendorff** [comment]
Yeah. Even with `position: sticky`, I had to do some damage to the document structure to get that to work as desired. :-\

**Comment by jorendorff** [comment]
In particular: before the sticky headings, I had nested sections.

But with sticky headings, that gave me multiple sticky headings stuck to the top of the viewport in a pile: the heading for 9.1.1 would appear on top of the heading for 9.1 which was on top of the heading for section 9.

I decided the sticky headings were worth it and completely revamped the document structure so there weren't nested `<section>`s anymore. Kind of a flip decision.
